__tests__/unit/_utils/generate-config.js

```diff
@@ -4,102 +4,128 @@
  */
 'use strict';
 
-const mockfs = require('../../../__mocks__/fs');
-const MOCK_PACKAGES = mockfs.__fsMockFiles();
+console.log = jest.fn();
```

Comment:
does this need to be here?

Reply:
we used to run jest with the `--silent` flag, but that has been dropped because sometimes you _want_ to log to the client (when debugging etc), but because this is a CLI app we log _a lot_ of stuff to the CLI. By adding this in here we can stop all the reporting from appearing in the test output

Reply:
you can then use `console.error` for example if you are debugging

Reply:
can also do `jest.mock('@springernature/util-cli-reporter');`, and I have just realised that I have done both! Think it is more obvious to mock the cli-reporter so will change it all to that (assuming there wasn't a reason I had both that I can't remember!)

Reply:
Oh, I've just realised what you meant! In this instance adding `jest.mock('@springernature/util-cli-reporter')` is more obvious for what is happening, but we still use `console.log = jest.fn()` in other places when testing error reporting

Reply:
hahaha this is excellent steam of consciousness

I hadn't thought that deeply about it if I'm honest with you - I thought you had perhaps just left debugging code in by accident.

I'm glad you've explained though, it is good to know for anything I do like this in future.